### PR TITLE
issue_813: added transliteration library

### PR DESCRIPTION
**Pull request by arslanbekova**

### CODE REVIEW

File: src/features/pageObjects/utils/pageObjectTemplate.js [hunk 1]
```diff
@@ -1,20 +1,11 @@
-import CyrillicToTranslit from "cyrillic-to-translit-js";
```
Comment (MariiaNebesnova) [inline src/features/pageObjects/utils/pageObjectTemplate.js:1]: please, remove it from package.json too

**Review by MariiaNebesnova — COMMENTED** [review]
please, add tests for `getClassName` with title includes some symbols that are expected to be transliterated by library

File: src/features/pageObjects/utils/pageObjectTemplate.js (hunk 1 above)
Comment (arslanbekova) [reply src/features/pageObjects/utils/pageObjectTemplate.js:1]: > please, remove it from package.json too

it's already here [https://github.com/jdi-testing/jdn-ai/pull/964/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L51](package.json 51 line)

**Comment by MariiaNebesnova** [comment]
please, commit updated package-lock

**Pull request merged**